### ISSUE TITLE
Design refactored manuscript download service

### ISSUE THREAD

**Issue by markpatton** [eclipse-pass/main#341]
The goal is to come up with a design for the service in sufficient detail that others can review and give feedback. To accomplish this goal, create a pull request that shows the design as a minimally implemented application. The application must compile, but does not need to run. The text of the pull request should give a high level explanation of the design and any differences from the original service.

Design:
- Compare the existing service to how it is actually being used.
  - Are all the features needed?
- Separate business logic from web request handling
- Provide skeleton code that should compile and show the general approach.
- Should either be a Spring Boot application or a Java web application
- Can be a single maven module or a multi-module if justified by the design

Infrastructure:
- Put in GitHub build actions. (Can copy from an existing project like pass-doi-service.)
- Figure out what dependencies you will need and add them to the pom
- Update the README to the extent possible such that it reflects the new design

**Comment by maggieolaya** [comment]
Traced Go code to find logic flow and created a rudimentary java skeleton. I left out the web handling for now, as I was just focused on the flow between the functions. By doing this, I traced where and when functions are called and created an outline for it in the java files. I created a PR for review of my initial structure: [PR](https://github.com/eclipse-pass/pass-download-service/pull/11#issue-1438061392)

**Comment by maggieolaya** [comment]
I restructured my files so that they match up with the java implementation branch. I also fixed some java syntax errors in my naming and in my code. This week I began looking into the web service aspect of the service and am figuring out which classes I want to have interacting with it. I also am in the midst of finding equivalent java imports to the ones in the GO files and figuring out which classes need which.[PR](https://github.com/eclipse-pass/pass-download-service/pull/12#issue-1447521964)
